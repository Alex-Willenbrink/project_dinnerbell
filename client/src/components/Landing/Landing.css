@import url('https://fonts.googleapis.com/css?family=Indie+Flower|Open+Sans:400,700');

.landing {
  display: flex;
  flex-direction: column;
  align-items: center;
  width: 100vw;
  height: 100vh;
  background-color: #3c8d41;
  position: relative;
}

.landing-images {
  width: 315px;
  margin-bottom: 7px;
  display: flex;
  flex-direction: row;
  justify-content: space-between;
}

.landing-title {
  font-family: 'Indie Flower', cursive;
  font-size: 5em;
  color: white;
}

.landing-description {
  font-family: 'Open Sans', sans-serif;
  color: white;
  position: absolute;
  top: 21%;
  z-index: 3;
}

.landing-description span {
  font-weight: bold;
}

.landing-img {
  width: 100%;
  object-fit: fill;
}

.landing-auth-container {
  display: flex;
  width: 200px;
  justify-content: space-between;
  top: 32%;
  position: absolute;
}

.landing-auth-container button {
  font-family: 'Open Sans', sans-serif;
  font-size: 16px;
  padding: 5px;
  padding-right: 10px;
  padding-left: 10px;
  border: 0.5px solid white;
  color: white;
  background-color: #3c8d41;
  border-radius: 5px;
}

/* Row 1 */

.landing-img-container-1 {
  width: 100px;
  height: 100px;
  background-image: url('https://www.bellamysorganic.com.au/blog/wp-content/uploads/2013/12/understanding-the-health-benefits-of-organic-food-1.jpg');
  background-size: cover;
  background-repeat: no-repeat;
  background-position: 50% 50%;
}

.landing-img-container-2 {
  width: 100px;
  height: 100px;
  background-image: url('http://cdn.blessthisstuff.com/imagens/stuff/sprig-organic-meals-3.jpg');
  background-size: cover;
  background-repeat: no-repeat;
  background-position: 50% 50%;
}

.landing-img-container-3 {
  width: 100px;
  height: 100px;
  background-image: url('http://www.singletonsseafoodshack.com/wp-content/uploads/2017/05/Restaurant-Eating.jpg');
  background-size: cover;
  background-repeat: no-repeat;
  background-position: 50% 50%;
}

/* Row 2 */

.landing-img-container-4 {
  width: 100px;
  height: 100px;
  background-image: url('http://roguehealthandfitness.com/wp-content/uploads/2015/02/organic.jpg');
  background-size: cover;
  background-repeat: no-repeat;
  background-position: 50% 50%;
}

.landing-img-container-5 {
  width: 100px;
  height: 100px;
  background-image: url('http://cdn.theatlantic.com/static/mt/assets/food/salad.jpg');
  background-size: cover;
  background-repeat: no-repeat;
  background-position: 50% 50%;
}

.landing-img-container-6 {
  width: 100px;
  height: 100px;
  background-image: url('http://readynutrition.com/wp-content/uploads/2014/04/shutterstock_151441463.jpg');
  background-size: cover;
  background-repeat: no-repeat;
  background-position: 50% 50%;
}

/* Row 3 */

.landing-img-container-7 {
  width: 100px;
  height: 100px;
  background-image: url('http://2zcb9n15d8i73ncvp0lwwrct.wpengine.netdna-cdn.com/wp-content/uploads/organic-whole-grain-oatmeal-pancakes-768x423.png');
  background-size: cover;
  background-repeat: no-repeat;
  background-position: 50% 50%;
}

.landing-img-container-8 {
  width: 100px;
  height: 100px;
  background-image: url('https://sunbasket.com/resources/img/home/how-it-works/three-recipe-bags-2.jpg');
  background-size: cover;
  background-repeat: no-repeat;
  background-position: 50% 50%;
}

.landing-img-container-9 {
  width: 100px;
  height: 100px;
  background-image: url('https://www.rd.com/wp-content/uploads/sites/2/2016/06/10-things-know-organic-food-farming.jpg');
  background-size: cover;
  background-repeat: no-repeat;
  background-position: 50% 50%;
}

.landing-img-cube {
  margin-top: 30px;
}
