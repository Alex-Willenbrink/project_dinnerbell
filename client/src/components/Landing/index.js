import React, { Component } from 'react';
import { Paper } from 'material-ui';

import './Landing.css';

const style = {
  height: 100,
  width: 100
};

const Landing = () => {
  return (
    <section className="landing">
      <h1 className="landing-title">dinnerbell</h1>
      <div className="landing-auth-container">
        <button>register</button>
        <button>login</button>
      </div>
      <div className="landing-images ">
        <Paper style={style} zDepth={2}>
          <div className="landing-img-container-1" />
        </Paper>
        <Paper style={style} zDepth={2}>
          <div className="landing-img-container-2" />
        </Paper>
        <Paper style={style} zDepth={2}>
          <div className="landing-img-container-3" />
        </Paper>
      </div>
      <div className="landing-images">
        <Paper style={style} zDepth={2}>
          <div className="landing-img-container-4" />
        </Paper>
        <Paper style={style} zDepth={2}>
          <div className="landing-img-container-5" />
        </Paper>
        <Paper style={style} zDepth={2}>
          <div className="landing-img-container-6" />
        </Paper>
      </div>
      <div className="landing-images">
        <Paper style={style} zDepth={2}>
          <div className="landing-img-container-7" />
        </Paper>
        <Paper style={style} zDepth={2}>
          <div className="landing-img-container-8" />
        </Paper>
        <Paper style={style} zDepth={2}>
          <div className="landing-img-container-9" />
        </Paper>
      </div>
    </section>
  );
};

export default Landing;

// <img
//   className="landing-img"
//   src="https://www.bellamysorganic.com.au/blog/wp-content/uploads/2013/12/understanding-the-health-benefits-of-organic-food-1.jpg"
// />
