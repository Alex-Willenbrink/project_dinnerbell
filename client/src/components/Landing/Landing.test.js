import MuiThemeProvider from "material-ui/styles/MuiThemeProvider";
import { configure } from "enzyme";
import Adapter from "enzyme-adapter-react-16";

configure({ adapter: new Adapter() });
import React from "react";
import { shallow, mount } from "enzyme";
import renderer from "react-test-renderer";
import Landing from "./index";
import { Paper } from "material-ui";

describe(Landing, () => {
  const landing = (
    <MuiThemeProvider>
      <Landing />
    </MuiThemeProvider>
  );
  const component = mount(landing);
  it("renders and matches our snapshot", () => {
    const component = renderer.create(landing);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
  it("has a class called landing", () => {
    expect(component.find(".landing").hasClass("landing")).toEqual(true);
  });
  it("has a title that has a class of landing-title", () => {
    expect(component.find("h1").hasClass("landing-title")).toEqual(true);
  });
  it("contains 9 paper components", () => {
    expect(component.find(Paper)).toHaveLength(9);
  });

  // it('adds another greeting when the add greeting function is called', () => {
  //   const before = component.find(HelloWorld).length;
  //   component.instance().addGreeting('Sample');
  //   const after = component.find(HelloWorld).length;
  //   expect(after).toBeGreaterThan(before);
  // });
  // it('removes a greeting from the list when the remove greeting function is called', () => {
  //   const before = component.find(HelloWorld).length;
  //   const removeMe = component.state('greetings')[0];
  //   component.instance().removeGreeting(removeMe);
  //   const after = component.find(HelloWorld).length;
  //   expect(after).toBeLessThan(before);
  // });
});
