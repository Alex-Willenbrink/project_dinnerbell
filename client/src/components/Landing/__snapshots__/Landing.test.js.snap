// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`() => {
  return (
    _react2.default.createElement("section", { className: "landing", __source: { fileName: _jsxFileName, lineNumber: 13 }, __self: undefined },
      _react2.default.createElement("h1", { className: "landing-title", __source: { fileName: _jsxFileName, lineNumber: 14 }, __self: undefined }, "Dinnerbell"),
      _react2.default.createElement("div", { className: "landing-images", __source: { fileName: _jsxFileName, lineNumber: 15 }, __self: undefined },
        _react2.default.createElement(_materialUi.Paper, { style: style, zDepth: 2, __source: { fileName: _jsxFileName, lineNumber: 16 }, __self: undefined },
          _react2.default.createElement("div", { className: "landing-img-container-1", __source: { fileName: _jsxFileName, lineNumber: 17 }, __self: undefined })),

        _react2.default.createElement(_materialUi.Paper, { style: style, zDepth: 2, __source: { fileName: _jsxFileName, lineNumber: 19 }, __self: undefined },
          _react2.default.createElement("div", { className: "landing-img-container-2", __source: { fileName: _jsxFileName, lineNumber: 20 }, __self: undefined })),

        _react2.default.createElement(_materialUi.Paper, { style: style, zDepth: 2, __source: { fileName: _jsxFileName, lineNumber: 22 }, __self: undefined },
          _react2.default.createElement("div", { className: "landing-img-container-3", __source: { fileName: _jsxFileName, lineNumber: 23 }, __self: undefined }))),


      _react2.default.createElement("div", { className: "landing-images", __source: { fileName: _jsxFileName, lineNumber: 26 }, __self: undefined },
        _react2.default.createElement(_materialUi.Paper, { style: style, zDepth: 2, __source: { fileName: _jsxFileName, lineNumber: 27 }, __self: undefined },
          _react2.default.createElement("div", { className: "landing-img-container-4", __source: { fileName: _jsxFileName, lineNumber: 28 }, __self: undefined })),

        _react2.default.createElement(_materialUi.Paper, { style: style, zDepth: 2, __source: { fileName: _jsxFileName, lineNumber: 30 }, __self: undefined },
          _react2.default.createElement("div", { className: "landing-img-container-5", __source: { fileName: _jsxFileName, lineNumber: 31 }, __self: undefined })),

        _react2.default.createElement(_materialUi.Paper, { style: style, zDepth: 2, __source: { fileName: _jsxFileName, lineNumber: 33 }, __self: undefined },
          _react2.default.createElement("div", { className: "landing-img-container-6", __source: { fileName: _jsxFileName, lineNumber: 34 }, __self: undefined }))),


      _react2.default.createElement("div", { className: "landing-images", __source: { fileName: _jsxFileName, lineNumber: 37 }, __self: undefined },
        _react2.default.createElement(_materialUi.Paper, { style: style, zDepth: 2, __source: { fileName: _jsxFileName, lineNumber: 38 }, __self: undefined },
          _react2.default.createElement("div", { className: "landing-img-container-7", __source: { fileName: _jsxFileName, lineNumber: 39 }, __self: undefined })),

        _react2.default.createElement(_materialUi.Paper, { style: style, zDepth: 2, __source: { fileName: _jsxFileName, lineNumber: 41 }, __self: undefined },
          _react2.default.createElement("div", { className: "landing-img-container-8", __source: { fileName: _jsxFileName, lineNumber: 42 }, __self: undefined })),

        _react2.default.createElement(_materialUi.Paper, { style: style, zDepth: 2, __source: { fileName: _jsxFileName, lineNumber: 44 }, __self: undefined },
          _react2.default.createElement("div", { className: "landing-img-container-9", __source: { fileName: _jsxFileName, lineNumber: 45 }, __self: undefined })))));




} renders and matches our snapshot 1`] = `
<section
  className="landing"
>
  <h1
    className="landing-title"
  >
    Dinnerbell
  </h1>
  <div
    className="landing-images"
  >
    <div
      style={
        Object {
          "WebkitTapHighlightColor": "rgba(0,0,0,0)",
          "backgroundColor": "#ffffff",
          "borderRadius": 2,
          "boxShadow": "0 3px 10px rgba(0, 0, 0, 0.16),
                 0 3px 10px rgba(0, 0, 0, 0.23)",
          "boxSizing": "border-box",
          "color": "rgba(0, 0, 0, 0.87)",
          "fontFamily": "Roboto, sans-serif",
          "height": 100,
          "muiPrepared": true,
          "transition": "all 450ms cubic-bezier(0.23, 1, 0.32, 1) 0ms",
          "width": 100,
        }
      }
    >
      <div
        className="landing-img-container-1"
      />
    </div>
    <div
      style={
        Object {
          "WebkitTapHighlightColor": "rgba(0,0,0,0)",
          "backgroundColor": "#ffffff",
          "borderRadius": 2,
          "boxShadow": "0 3px 10px rgba(0, 0, 0, 0.16),
                 0 3px 10px rgba(0, 0, 0, 0.23)",
          "boxSizing": "border-box",
          "color": "rgba(0, 0, 0, 0.87)",
          "fontFamily": "Roboto, sans-serif",
          "height": 100,
          "muiPrepared": true,
          "transition": "all 450ms cubic-bezier(0.23, 1, 0.32, 1) 0ms",
          "width": 100,
        }
      }
    >
      <div
        className="landing-img-container-2"
      />
    </div>
    <div
      style={
        Object {
          "WebkitTapHighlightColor": "rgba(0,0,0,0)",
          "backgroundColor": "#ffffff",
          "borderRadius": 2,
          "boxShadow": "0 3px 10px rgba(0, 0, 0, 0.16),
                 0 3px 10px rgba(0, 0, 0, 0.23)",
          "boxSizing": "border-box",
          "color": "rgba(0, 0, 0, 0.87)",
          "fontFamily": "Roboto, sans-serif",
          "height": 100,
          "muiPrepared": true,
          "transition": "all 450ms cubic-bezier(0.23, 1, 0.32, 1) 0ms",
          "width": 100,
        }
      }
    >
      <div
        className="landing-img-container-3"
      />
    </div>
  </div>
  <div
    className="landing-images"
  >
    <div
      style={
        Object {
          "WebkitTapHighlightColor": "rgba(0,0,0,0)",
          "backgroundColor": "#ffffff",
          "borderRadius": 2,
          "boxShadow": "0 3px 10px rgba(0, 0, 0, 0.16),
                 0 3px 10px rgba(0, 0, 0, 0.23)",
          "boxSizing": "border-box",
          "color": "rgba(0, 0, 0, 0.87)",
          "fontFamily": "Roboto, sans-serif",
          "height": 100,
          "muiPrepared": true,
          "transition": "all 450ms cubic-bezier(0.23, 1, 0.32, 1) 0ms",
          "width": 100,
        }
      }
    >
      <div
        className="landing-img-container-4"
      />
    </div>
    <div
      style={
        Object {
          "WebkitTapHighlightColor": "rgba(0,0,0,0)",
          "backgroundColor": "#ffffff",
          "borderRadius": 2,
          "boxShadow": "0 3px 10px rgba(0, 0, 0, 0.16),
                 0 3px 10px rgba(0, 0, 0, 0.23)",
          "boxSizing": "border-box",
          "color": "rgba(0, 0, 0, 0.87)",
          "fontFamily": "Roboto, sans-serif",
          "height": 100,
          "muiPrepared": true,
          "transition": "all 450ms cubic-bezier(0.23, 1, 0.32, 1) 0ms",
          "width": 100,
        }
      }
    >
      <div
        className="landing-img-container-5"
      />
    </div>
    <div
      style={
        Object {
          "WebkitTapHighlightColor": "rgba(0,0,0,0)",
          "backgroundColor": "#ffffff",
          "borderRadius": 2,
          "boxShadow": "0 3px 10px rgba(0, 0, 0, 0.16),
                 0 3px 10px rgba(0, 0, 0, 0.23)",
          "boxSizing": "border-box",
          "color": "rgba(0, 0, 0, 0.87)",
          "fontFamily": "Roboto, sans-serif",
          "height": 100,
          "muiPrepared": true,
          "transition": "all 450ms cubic-bezier(0.23, 1, 0.32, 1) 0ms",
          "width": 100,
        }
      }
    >
      <div
        className="landing-img-container-6"
      />
    </div>
  </div>
  <div
    className="landing-images"
  >
    <div
      style={
        Object {
          "WebkitTapHighlightColor": "rgba(0,0,0,0)",
          "backgroundColor": "#ffffff",
          "borderRadius": 2,
          "boxShadow": "0 3px 10px rgba(0, 0, 0, 0.16),
                 0 3px 10px rgba(0, 0, 0, 0.23)",
          "boxSizing": "border-box",
          "color": "rgba(0, 0, 0, 0.87)",
          "fontFamily": "Roboto, sans-serif",
          "height": 100,
          "muiPrepared": true,
          "transition": "all 450ms cubic-bezier(0.23, 1, 0.32, 1) 0ms",
          "width": 100,
        }
      }
    >
      <div
        className="landing-img-container-7"
      />
    </div>
    <div
      style={
        Object {
          "WebkitTapHighlightColor": "rgba(0,0,0,0)",
          "backgroundColor": "#ffffff",
          "borderRadius": 2,
          "boxShadow": "0 3px 10px rgba(0, 0, 0, 0.16),
                 0 3px 10px rgba(0, 0, 0, 0.23)",
          "boxSizing": "border-box",
          "color": "rgba(0, 0, 0, 0.87)",
          "fontFamily": "Roboto, sans-serif",
          "height": 100,
          "muiPrepared": true,
          "transition": "all 450ms cubic-bezier(0.23, 1, 0.32, 1) 0ms",
          "width": 100,
        }
      }
    >
      <div
        className="landing-img-container-8"
      />
    </div>
    <div
      style={
        Object {
          "WebkitTapHighlightColor": "rgba(0,0,0,0)",
          "backgroundColor": "#ffffff",
          "borderRadius": 2,
          "boxShadow": "0 3px 10px rgba(0, 0, 0, 0.16),
                 0 3px 10px rgba(0, 0, 0, 0.23)",
          "boxSizing": "border-box",
          "color": "rgba(0, 0, 0, 0.87)",
          "fontFamily": "Roboto, sans-serif",
          "height": 100,
          "muiPrepared": true,
          "transition": "all 450ms cubic-bezier(0.23, 1, 0.32, 1) 0ms",
          "width": 100,
        }
      }
    >
      <div
        className="landing-img-container-9"
      />
    </div>
  </div>
</section>
`;
